compiler/aarch64/codegen/UnaryEvaluator.cpp

```diff
@@ -81,17 +81,36 @@ TR::Register *OMR::ARM64::TreeEvaluator::lnegEvaluator(TR::Node *node, TR::CodeG
 	return OMR::ARM64::TreeEvaluator::unImpOpEvaluator(node, cg);
 	}
 
+static TR::Register *commonAbsEvaluator(TR::Node *node, TR::CodeGenerator *cg)
```

Comment:
Can you call this `commonIntegerAbsEvaluator` to make it clear this is for integer operations?